hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/v2/components/tables/insights/deletePendingKeysTable.tsx

```diff
@@ -0,0 +1,194 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements.  See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership.  The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import React from 'react';
+import { AxiosError } from 'axios';
+import Table, {
+  ColumnsType,
+  TablePaginationConfig
+} from 'antd/es/table';
+import { ValueType } from 'react-select';
+
+import Search from '@/v2/components/search/search';
+import SingleSelect, { Option } from '@/v2/components/select/singleSelect';
+import ExpandedPendingKeysTable from '@/v2/components/tables/insights/expandedPendingKeysTable';
```

Comment:
When I first accessed the Insights page and clicked on the "Keys Pending for Deletion" tab, I encountered an error pop-up. However, this issue occurs only with this specific tab, not the others.

<img width="2058" alt="image" src="https://github.com/user-attachments/assets/b0cdd322-4262-4ad4-91f5-5b0efc6ddf3f">

Reply:
Addressed in [8add6c2](https://github.com/apache/ozone/pull/7327/commits/8add6c2fccf9b0e999fac47ef0a65fa19e996b2c)